test/test.js

```diff
@@ -13,369 +12,360 @@ function isPromise (x) {
   return x && typeof x.then === 'function'
 }
 
-function testBaseFunctionality() {
```

Comment:
why did we remove this?

Reply:
Just changed how tests are executed when changing the test runner. Want me to switch it back?

Reply:
yeah, we want the whole suite to run in both envs. looks like only 1 test runs in prod